components/input-group/src/InputGroup.js

```diff
@@ -0,0 +1,93 @@
+/** @jsx jsx */
+
+import React, { Children, cloneElement } from 'react';
+import PropTypes from 'prop-types';
+import { jsx } from '@westpac/core';
+import { useFormContext } from '@westpac/form';
+
+// ==============================
+// Component
+// ==============================
+
+/**
+ * Input Group: Styled input fields with attached addons. Addons can be text ($, %, .00 etc) or form controls (buttons or select inputs).
+ */
+export const InputGroup = ({ size, children, ...props }) => {
+	const formContext = useFormContext();
+
+	const inputGroupSize = (formContext && formContext.size) || size;
+
+	const childrenWithProps = Children.map(children, child =>
+		cloneElement(child, { size: inputGroupSize })
+	);
```

Comment:
Need to look into doing this via context. Would probably involve creating wrappers around the other gel components this is consuming. Leave for now and we can discuss with TM later.

Reply:
Hey @florettewest this was already done in the previous PR you closed